### PR TITLE
Test the models in the account app.

### PR DESCRIPTION
**Pull request by sabina489**
OTP-time is valid or invalid was tested and also tested validated otp reset is valid or invalid . Also phone is valid or invalid was tested.

### CODE REVIEW

File: accounts/test/test_models.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    def UserNew(self,phone="9821053490"):
+        return User.objects.create(phone=phone)
+    # self.otp_reset_expiry
+    def test_is_otp_time_valid(self):
+        time_now = timezone.datetime.now()
+        time_now = timezone.make_aware(time_now, timezone.utc)
+        time_1_min_after_now = time_now+timezone.timedelta(minutes=1)
+        usernew = self.UserNew()
+        usernew.otp_reset_expiry = time_1_min_after_now
+        # print('******', usernew.is_otp_reset_time_valid)
```
Comment (shovan777) [inline accounts/test/test_models.py:22]: Remove the comments of print please @sabina489.

**Review by shovan777 — CHANGES_REQUESTED** [review]
Please make the changes as the comments.

File: accounts/test/test_models.py (hunk 1 above)
Comment (sabina489) [reply accounts/test/test_models.py]: The changes as per comments have been made in [cfcb9f3](https://github.com/prokuranepal/miras-backend/pull/137/commits/cfcb9f3a3a71882f45221be3ef42ce6284addc9f). please review @shovan777

**Review by sabina489 — COMMENTED** [review]
Changes have been made.

**Review by shovan777 — APPROVED** [review]
good job :fire:

**Pull request merged**